common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -27,15 +29,46 @@ def setUp(self):
         """
         super(AccountSettingsPageTest, self).setUp()
 
-        AutoAuthPage(self.browser, username=self.USERNAME, password=self.PASSWORD, email=self.EMAIL).visit()
+        self.user_id = AutoAuthPage(
+            self.browser, username=self.USERNAME, password=self.PASSWORD, email=self.EMAIL
+        ).visit().get_user_id()
 
+    def visit_account_settings_page(self):
```

Comment:
Nit: is there any reason to not call visit_account_settings_page in the setup method? It looks like you have to call this method at the beginning of every test method.